test/runtests.jl

```diff
@@ -2872,3 +2872,22 @@ end
     @test triu(Z, 2) === Z
     @test tril(Z, 2) === Z
 end
+
+@testset "sqrt/cbrt" begin
+    F = Fill(4, 4, 4)
+    @test sqrt(F)^2 ≈ F
```

Comment:
We should add tests that check it matches dense: eg add `@test sqrt(F) ≈ sqrt(Matrix(F))`

Reply:
I've added some tests that check that these are approximately equal within some tolerance (typically around `sqrt(1e-16)` or `cbrt(1e-16)`)